src/index.js

```diff
@@ -20,15 +17,19 @@ import App from './App';
 import registerServiceWorker from './registerServiceWorker';
 
 ReactDOM.render(
-  <Router>
-    <App>
-      <Switch>
-        <Route exact path={process.env.BASE_PATH} component={Main} />
-        {/* New examples here */}
-        <Redirect exact from="*" to={process.env.BASE_PATH} />
-      </Switch>
-    </App>
-  </Router>,
+  <div>
```

Comment:
This div is not necessary, right?

Reply:
Yep, left over because I wanted to log the BASE_PATH - shall remove